cmd/pebble/cmd_enter.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/canonical/pebble/internal/logger"
+	"github.com/jessevdk/go-flags"
+)
+
+const shortEnterHelp = "Run pebble command in new pebble environment"
+const longEnterHelp = `
```

Comment:
Okay, here is the proposal
Short help: "Run subcommand under a container environment"

Long help:
```
The enter command facilitates the use of Pebble as an entrypoint for containers.
When used without a subcommand, it mimics the behavior of the run command
alone, while if used with a subcommand it runs that subcommand in the most
appropriate environment taking in account its purpose.

These subcommands are currently supported:

  help      (1)(2)
  version   (1)(2)
  plan      (1)(2)
  services  (1)(2)
  ls        (1)(2)
  exec      (2)
  start     (3)
  stop      (3)

(1) Services are not started.
(2) No logs on stdout unless -v is used.
(3) Services continue running after the subcommand succeeds.
```

Reply:
looks good.

nitpick:

```
< taking in account 
> taking into account
```